triggers.bat

```diff
@@ -0,0 +1 @@
+dotnet fsi triggers.fsx -- %*
```

Comment:
Ty soubory bat a sh bych sem nedával. Místo toho bych to spíš napsal do README. K čemu to je jak se to spouští a co to vrací za výstup. Asi bych to taky umístil do podložky, Documentation?

Reply:
hotovo, prosím o review